_releases/2023-06-20-may-2023.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+category: releases
+title: May 2023 Release Notes
+filename-custom: may-2023
+last_modified_at: 
+sidenav: about_us
+release-year: "2023"
+description: "Release notes for May 2023, including several improvements and one fix."
+---
+## Improvements
+
+* **Results Page Redesign**: We’ve continued progress on updating our frontend components.
+  * We added a button to the Admin Center to allow users to preview the new results page design. To find it, log into the [Admin Center](https://search.usa.gov/sites) > Display > Redesigned Display.
+  * We began work to update the user interface to manage the redesigned display settings in the Admin Center.
+  * We added search term highlighting for the result snippet text.
+  * We’ve added pagination in our results page redesign. We’ll show up to 500 pages of results.
```

Comment:
I try to avoid smart quotes because they sometimes cause problems for ?older? automated page readers and scrapers.

Reply:
straight-quotes-ified